src/app/auth/auth-config.module.ts

```diff
@@ -0,0 +1,55 @@
+import { NgModule } from '@angular/core';
+import { HttpClient, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { AuthInterceptor, AuthModule, OpenIdConfiguration, StsConfigHttpLoader, StsConfigLoader } from 'angular-auth-oidc-client';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { AuthOptions } from '@congarevenuecloud/core';
+import { config$ } from '../../main';
+import { AuthorizationGuard } from './auth.guard';
+import { AppComponent } from '../app.component';
+
+export const httpLoaderFactory = () => {
+
+  const authOptions$: Observable<OpenIdConfiguration> = config$.pipe(
+    map((authOptions: AuthOptions) => {
+      return {
+        authority: authOptions.authEndpoint,
+        redirectUrl: 'https://localhost:3000',
```

Comment:
We shouldn't hardcode redirect url to user localhost. Please update to `window.location.origin + window.location.pathname`